src/orca/services/sevenbridges/ops.py

```diff
@@ -47,23 +24,25 @@ class SevenBridgesOps:
     """
 
     client: Api
-    project: Optional[str] = None
+    project: str
```

Comment:
This makes sense. Most interactions with SevenBridgesOps is going to be around a project (as it pertains to the Hook).  As you've mentioned previously, the beauty of this implementation is that you can still interact with all of the seven bridges API.

```
ops.client.projects.query()
```